pkgs/development/python-modules/amazon-s3-plugin-for-pytorch/default.nix

```diff
@@ -0,0 +1,66 @@
+{ buildPythonPackage
+, aws-sdk-cpp
+, cmake
+, curl
+, fetchFromGitHub
+, lib
+, pybind11
+, pytorch
+, zlib
+}:
+
+let
+  aws-sdk-cpp-s3-transfer = (aws-sdk-cpp.override {
+    apis = ["s3" "transfer"];
+    customMemoryManagement = false;
+  }).overrideAttrs (oldAttrs: {
+
+    # Fixes downstream issue with dependent CMake configuration
+    # See https://github.com/NixOS/nixpkgs/issues/70075#issuecomment-1019328864
+    postPatch = oldAttrs.postPatch + ''
+      substituteInPlace cmake/AWSSDKConfig.cmake \
+        --replace "\''${AWSSDK_DEFAULT_ROOT_DIR}/\''${AWSSDK_INSTALL_INCLUDEDIR}" "\''${AWSSDK_INSTALL_INCLUDEDIR}"
+    '';
```

Comment:
This is related to https://github.com/NixOS/nixpkgs/issues/70075#issuecomment-1019328864 and probably needs to be fixed more directly in `aws-sdk-cpp`.